invenio_oauth2server/templates/invenio_oauth2server/settings/client_view.html

```diff
@@ -68,11 +69,11 @@
             {{ _('example request') }}
           </a>):
           <ul>
-            <li>response_type (required, use <code>code</code> or <code>token</code>)</li>
-            <li>client_id (required)</li>
-            <li>scope (required, space separate list of scopes)</li>
-            <li>redirect_uri (required, URL encoded)</li>
-            <li>state (recommended, for CSRF protection)</li>
+            <li>response_type {{ _('(required, use') }} <code>code</code> {{ _('or') }} <code>token</code>)</li>
+            <li>client_id {{ _('(required)</li>
+            <li>scope {{ _('(required, space separate list of scopes)') }}</li>
+            <li>redirect_uri {{ _('(required, URL encoded)') }}</li>
+            <li>state {{ _('(recommended, for CSRF protection)') }}</li>
```

Comment:
could you double-check if this would work?

Reply:
why do you need to include in the translations the `()`?
```suggestion
            <li>response_type {{ (_('required, use') }} <code>code</code> {{ _('or') }} <code>token</code>)</li>
            <li>client_id {{ (_('required'))</li>
            <li>scope {{ (_('required, space separate list of scopes')) }}</li>
            <li>redirect_uri {{ (_('required, URL encoded')) }}</li>
            <li>state {{ (_('recommended, for CSRF protection')) }}</li>
```